cli/src/config/misc.toml

```diff
@@ -7,3 +7,6 @@
 [ui]
 pager = { command = ["less", "-FRX"], env = { LESSCHARSET = "utf-8" } }
 log-word-wrap = false
+
+[snapshot]
+max-new-file-size = 1048576
```

Comment:
I would love if there was a way to specify "1MiB" here

Reply:
Implemented in a second commit.